src/services/SubmissionConfirmationService.ts

```diff
@@ -67,7 +67,7 @@ export class SubmissionConfirmationService implements OnInit {
     private sendConfirmationEmail(pendingEntry: PendingEntryConfirmationModel): Promise<string> {
         const baseUrl = process.env.BASE_URL;
         const confirmationUrl = `${baseUrl}/processSubmission?uid=${pendingEntry.confirmationUid}`;
-        const body = `Please click the link below to confirm your submission. This link will expire in 20 minutes\n${confirmationUrl}`;
+        const body = `Please click the link below to confirm your submission. This link will expire in 20 minutes. \n${confirmationUrl}`;
```

Comment:
remove the space between `.` and `/n`
```suggestion
        const body = `Please click the link below to confirm your submission. This link will expire in 20 minutes.\n${confirmationUrl}`;
```